dfsclient/src/main/java/org/hpccsystems/dfs/client/FileUtility.java

```diff
@@ -2140,7 +2140,10 @@ public static JSONArray run(String[] args)
                 System.out.println("    otel.metrics.exporter: "+ System.getProperty("otel.metrics.exporter"));
                 System.out.println("    OTEL_METRICS_EXPORTER Env var: " + System.getenv("OTEL_METRICS_EXPORTER"));
 
-                OpenTelemetry otel = AutoConfiguredOpenTelemetrySdk.initialize().getOpenTelemetrySdk();
+                if (!org.hpccsystems.ws.client.utils.Utils.isOtelJavaagentUsed())
+                {
+                    AutoConfiguredOpenTelemetrySdk.initialize().getOpenTelemetrySdk();

```

Comment:
is "otelInitialized" in L:2149 accurate?
It seems to assume otel has been initialized even if we don't know for sure.

Reply:
Good question, I am using that to prevent attempting to setup Otel multiple times. (Run can be called more than once.) So, that should still work as intended.

Reply:
per offline convo let's refactor the variable name.